accounting/accounting.go

```diff
@@ -250,44 +252,45 @@ func (accounting *State) AddTransaction(txnr *idb.TxnRow) (err error) {
 		accounting.updateAccountType(stxn.Txn.Sender, string(ktype))
 	}
 
-	accounting.updateAlgo(stxn.Txn.Sender, -int64(stxn.Txn.Fee))
-	accounting.updateAlgo(accounting.feeAddr, int64(stxn.Txn.Fee))
+	accounting.updateAlgo(stxn.Txn.Sender, -int64(stxn.Txn.Fee.Raw))
+	accounting.updateAlgo(accounting.feeAddr, int64(stxn.Txn.Fee.Raw))
 
-	if stxn.SenderRewards != 0 {
+	if stxn.SenderRewards != (basics.MicroAlgos{}) {
```

Comment:
ahh.. you're aware of `MicroAlgos.IsZero()`, right ?

Reply:
I was not. Changed it.